packages/block-editor/src/layouts/utils.js

```diff
@@ -17,7 +17,48 @@ export function appendSelectors( selectors, append = '' ) {
 		.split( ',' )
 		.map(
 			( subselector ) =>
-				`.editor-styles-wrapper ${ subselector } ${ append }`
+				`.editor-styles-wrapper ${ subselector }${
+					append ? ` ${ append }` : ''
+				}`
```

Comment:
Oh, thanks for the link to the other function. I think that one might be a bit more heavy-handed than this one for the use case of constructing Layout styles, as that one appears to parse all the CSS rules in order to then wrap the selectors? For the Layout support, I think we can get away with something a little lighter.

The reason for this change, though, is a bit more subtle. The function already supports being called without an `append` param (it defaults to an empty string) and the Flex layout is currently calling it that way on `trunk`, but without this change, it returns additional whitespace at the end of the string. There weren't any unit tests for this function, so I think it was never caught that the function would return unexpected whitespace, but I ran into it when adding tests for the new `getBlockGapCSS` function.

Happy to rework things if you think it's better to change the approach, though!

Reply:
True, the other one takes the whole style rule instead of just the selector, and we probably want to use the same approach here whether we have selectors to append or not. Good catch with the bug though! And thanks for adding in tests 🙇